### ISSUE TITLE
Installing gulp via npm after cloning the repository gives an error

### ISSUE THREAD

**Issue by nicola** [mozbrick/brick#197]
I have a fresh install of Mavericks in front of me, this is what I did:
1. Installed node (v0.10.29)
2. Installed bower (1.3.8)
3. git cloned this repository

Now, when installing `gulp` in that folder I get:

```
localhost:brick ngreco$ npm install -g gulp
npm ERR! addLocal Could not install gulp
npm ERR! Error: ENOENT, open 'gulp/package.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 13.2.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "-g" "gulp"
npm ERR! cwd /Users/ngreco/Proj/mozilla/brick/brick
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.5.0-alpha-1
npm ERR! path gulp/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/ngreco/Proj/mozilla/brick/brick/npm-debug.log
npm ERR! not ok code 0
```

However if I switch path and I execute the same command it succeeds. Maybe an issue with the package.json or a folder issue since it may go and look into the folder gulp?

**Comment by nicola** [comment]
My second suggestion is correct. By renaming the folder `gulp`, installing gulp succeeds.
Hence we need to change name of that folder.

**Comment by potch** [comment]
Fixed by #223.
